benchmarkapps/Benchmarks/benchmarks.json

```diff
@@ -8,12 +8,15 @@
     },
     "Source": {
       "Repository": "https://github.com/aspnet/routing.git",
-      "BranchOrCommit": "dev",
+      "BranchOrCommit": "kichalla/added.benchmark.for.dispatcher",
       "Project": "benchmarkapps/Benchmarks/Benchmarks.csproj"
     },
     "Port": 8080
   },
   "PlaintextRouting": {
     "Path": "/plaintext"
+  },
+  "PlaintextDispatcher": {
```

Comment:
So if I'm understanding correctly, this will automatically pass the 'scenario' name as a command line arg?

Reply:
Yes, that's correct.